Documentation/MigrateToComposer/Requirements.rst

```diff
@@ -64,8 +88,52 @@ a later step. We will use :file:`public/` in our example.
 ..  todo: What does refactor concrete mean?
 
 If you do not have such a web root directory, you will have to refactor your
```

Comment:
typo3conf and typo3temp should be removed in the folder structure above, and vendor added.

Reply:
Yup, okay.